pebblo/app/service/prompt_service.py

```diff
@@ -143,15 +148,13 @@ def process_request(self):
             logger.debug("AI App prompt request processing started")
 
             # getting prompt data
-            prompt_data = self._fetch_classified_data(
-                self.data.get("prompt", {}).get("data"), input_type="prompt"
-            )
-
+            prompt_data = self.data.get("prompt", {})
+            logger.info(f"Pronpt Data {prompt_data}")
             is_prompt_gov_enabled = self.data.get("prompt", {}).get(
-                "prompt_gov_enabled", False
+                "promptGovEnabled", None
```

Comment:
As we are changing from `prompt_gov_enabled` to `promptGovEnabled`, we need to support `prompt_gov_enabled` for backward compatibility.